src/lib.rs

```diff
@@ -61,7 +61,7 @@ mod errors;
 mod identity;
 mod immutable_data;
 pub mod mutable_data;
-mod public_key;
+pub mod public_key;
```

Comment:
Why do we need to make this module public? The contents of it is re-exported as `pub use public_key::PublicKey`, so I'd probably just leave it as private.